lib/db.js

```diff
@@ -92,24 +92,45 @@ class LimitDBRedis extends EventEmitter {
     this.buckets[key] = utils.buildBucket(bucket);
   }
 
-  bucketKeyConfig(type, key) {
-    const fromCache = type.overridesCache.get(key);
+  paramsToBucketConfig(params) {
+    const bucketConfig = {};
+
+    if (params && params.configOverride) {
+      if (params.configOverride.bucketSize) {
+        bucketConfig.size = params.configOverride.bucketSize;
+      }
+      if (params.configOverride.refillRateMsPerToken) {
```

Comment:
This one can be zero, we use it for fixed size buckets in some cases and would make sense we also support it as an override.